project-koku/masu
# Provider remove summary data

Addresses #274 

Removing OCP and AWS summary data when data expires or when a provider is deleted.

**Testing**
1.  Add stand alone OCP provider, OCP (and AWS) for OCP-on-AWS.  Ingest relavent data
2. Verify the koku API to show initial datapoints for the providers
3.  Remove the AWS provider.  Verify OCP-on-AWS endpoint has no data as well as AWS endpoints
4. Remove my-ocp-cluster-2, verify my-ocp-cluster-1 remains
5. Verify OCP endpoitns and verify that my-ocp-cluster-2 is gone.
6.  Remove the last provider, my-ocp-cluster-1 and verify all OCP APIs return no data

**Test Results**
[provider_remove_ut.txt](https://github.com/project-koku/masu/files/3157694/provider_remove_ut.txt)
